website/settings/defaults.py

```diff
@@ -1934,3 +1934,10 @@ class CeleryConfig:
     'googleGroup': 'https://groups.google.com/forum/#!forum/openscienceframework',
     'github': 'https://www.github.com/centerforopenscience',
 }
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
```

Comment:
Drive-by: We don't have memcached set up. We'll need to use the Django database backend for now.

Reply:
I mean we don't have the [memcached](https://www.memcached.org/) service deployed.

So rather than relying on new infrastructure, we should just use our existing DB for now.